src/main.mjs

```diff
@@ -2,37 +2,41 @@ import { google } from 'googleapis';
 import { WebClient } from '@slack/web-api';
 import dayjs from 'dayjs';
 
-const getLuckyWinner = async (today: string) => {
+const getLuckyWinner = async (today, jsonSecret) => {
+
+  const serviceAccountEmail = jsonSecret?.client_email || process.env.SERVICE_ACCOUNT_EMAIL;
+  const serviceAccountKey = jsonSecret?.private_key || process.env.SERVICE_ACCOUNT_KEY || '';
+
+  console.log(`usingAccountEmail: ${serviceAccountEmail}`);
 
-  const serviceAccountEmail = process.env.SERVICE_ACCOUNT_EMAIL;
-  const serviceAccountKey: string = process.env.SERVICE_ACCOUNT_KEY || '';
-  const key = Buffer.from(serviceAccountKey, 'base64').toString('binary');
```

Comment:
This was causing a 403 when passing around the account key when I was grabbing stuff from secrets manager